docs/docs/aztec/developer/wallet-providers/account-abstraction.md

```diff
@@ -1,3 +1,113 @@
 # Account Abstraction
 
-Discuss account abstraction concepts.
\ No newline at end of file
+**Every account in Aztec is a smart contract** which defines the rules for whether a transaction is or not valid. This allows implementing different schemes for transaction signing, nonce management, and fee payments. However, encryption and nullifying keys, which are specific to private blockchains, are still enshrined at the protocol level.
+
+## Background
+
+We'll start with the mandatory "what is AA" section that every single article on the topic has, so you can skip this if you're familiar with the topic.
+
+### What is account abstraction?
+
+We'll refer to AA as the _ability to set the validity conditions of a transaction programmatically_ ([source](https://fuel-labs.ghost.io/account-abstraction-for-everyone-else/)). [Starknet](https://docs.starknet.io/documentation/architecture_and_concepts/Account_Abstraction/introduction/#account_abstraction) goes one step further and splits AA into three different components: 
+- Signature abstraction (defining when a signature is accepted)
+- Fee abstraction (paying fees)
+- Nonce abstraction (replay protection and ordering)
+
+In most AA schemes, the identity of a user is no longer represented by a keypair but by a contract, often called a smart contract wallet or account contract. This contract receives tx payloads which are validated with custom logic, and then interpreted as actions to execute, like calling into another contract.
+
+The benefits of AA are multiple. We're not going to reiterate them all here, but they include social recovery, MFA, batching, session keys, sponsored txs, fee payment in kind, supporting key schemes from different realms, etc. Read the articles from [Argent](https://www.argent.xyz/blog/part-3-wtf-is-account-abstraction/) or [Ethereum.org](https://ethereum.org/en/roadmap/account-abstraction/) for more detailed info.
+
+### Implementing at protocol vs application layer
+
+Account abstraction can be implemented at the application layer of a network using smart accounts and meta-transactions. The tx being sent to the network is still an Ethereum tx, but its payload is interpreted as a "transaction execution request" that is validated and run by the smart contract wallet. 
```

Comment:
> The tx being sent to the network is still an Ethereum tx...

It may not be clear prior to this statement that we are talking about AA on Ethereum L1 here. It might be best to say "For AA on Ethereum L1, a meta-transaction is still an Ethereum tx, but..."